geometry_msgs/Point new_coordinate
---
bool success
string message
